### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by RobertoCarrilloAvila**
# Description

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,6 @@
     "@react-google-maps/api": "^2.19.2",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
```
Comment (afmicc) [inline package.json:9]: this could be on dev dependencies too

File: src/__mocks__/hooks/useContentView.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
+export const mockGoTo = jest.fn();
```
Comment (afmicc) [inline src/__mocks__/hooks/useContentView.js:1]: as we already talked. I recommend to move this to each test file

File: src/components/CreateTarget/CreateTarget.test.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    },
+  ],
+}));
+
+test('has radius input', () => {
+  render(
+    <MapContextProvider>
+      <CreateTarget />
+    </MapContextProvider>,
+    { wrapper: BrowserRouter },
```
Comment (afmicc) [inline src/components/CreateTarget/CreateTarget.test.js:83]: as we talked, I'd recommend creating a new component to wrap all dependencies in one place

File: src/components/CreateTarget/CreateTarget.test.js [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+  );
+
+  const topic = screen.getByRole('combobox');
+
+  expect(topic).toBeInTheDocument();
+  await waitFor(() => expect(screen.getAllByRole('option')).toHaveLength(3));
+});
+
+jest.mock('hooks/useContentView');
```
Comment (afmicc) [inline src/components/CreateTarget/CreateTarget.test.js:116]: as we talked, I'd have all mocks placed together

File: src/components/CreateTarget/CreateTarget.test.js [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+      <CreateTarget />
+    </MapContext.Provider>,
+    { wrapper: BrowserRouter },
+  );
+
+  const user = userEvent.setup();
+  const radius = screen.getByRole('spinbutton');
+  const title = screen.getByRole('textbox');
+  const topic = screen.getByRole('combobox');
+  const submitButton = screen.getByRole('button');
```
Comment (afmicc) [inline src/components/CreateTarget/CreateTarget.test.js:140]: 🚀

File: src/services/__tests__/targetsService.test.js [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+describe('create target', () => {
+  const createRequest = {
+    title: 'new target',
+    radius: 27384.4,
+    topic_id: 2,
+    latitude: -94.5566,
+    longitude: -94.5566,
+  };
+  const createresponse = { target: createRequest };
```
Comment (afmicc) [inline src/services/__tests__/targetsService.test.js:16]:
```suggestion
  const createResponse = { target: createRequest };
```

**Review by marcio-b — APPROVED** [review]

File: src/components/About/About.test.js [hunk 7]
```diff
@@ -2,6 +2,23 @@ import { render, screen } from '@testing-library/react';
...
 
+jest.mock('hooks/useContentView', () => () => ({
+  displayedComponent: 'Home',
+  goTo: jest.fn(),
+  isMapVisible: false,
+  setIsMapVisible: jest.fn(),
+  navbarColor: 'blue',
+  setNavbarColor: jest.fn(),
+  navbarLeftButton: 'back',
+  setNavbarLeftButton: jest.fn(),
```
Comment (afmicc) [inline src/components/About/About.test.js:13]: NIT: you can omit and just mock what you use inside the component tested

File: src/components/CreateTarget/CreateTarget.test.js [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+  navbarColor: 'blue',
+  setNavbarColor: jest.fn(),
+  navbarLeftButton: 'back',
+  setNavbarLeftButton: jest.fn(),
+});
+
+jest.mock('hooks/useContentView', () => useContentView);
+
+const CreateTargetHarness = ({ values }) => (
+  <MapContext.Provider value={values}>
```
Comment (afmicc) [inline src/components/CreateTarget/CreateTarget.test.js:36]: you could have a name conflict here, at line 13 you define a const `values` too

Comment (afmicc) [reply src/components/CreateTarget/CreateTarget.test.js:36]: if you always pass the same values, you can omit to receive it by params
```suggestion
const CreateTargetHarness = () => (
  <MapContext.Provider value={values}>
```

**Review by afmicc — APPROVED** [review]

**Pull request merged**